lib/tasks/demo.rake

```diff
@@ -660,6 +664,42 @@ desc 'Backup the database'
     return histories
   end
 
+  def demo_populate_vaccines(today, days_ago, existing_patients)
+    printf("Generating vaccinations...")
+    vaccines = []
+    histories = []
+    patient_ids = existing_patients.pluck(:id).sample(existing_patients.count * rand(15..25) / 100)
+    patient_ids.each_with_index do |patient_id, index|
+      printf("\rGenerating vaccine #{index+1} of #{patient_ids.length}...")
+      changes_ts = create_fake_timestamp(today, today)
```

Comment:
can we rename this to `vaccine_ts` for consistency

Reply:
Done: https://github.com/SaraAlert/SaraAlert/pull/721/commits/66e9a6d482854054a97a2f0ac8c1ec4c0c6af3ca#diff-9bfc38ef71b77112eb91f77e35668d17dfea639c79e19de03900b201ad44b8efR674